.github/workflows/pull-request.yml

```
@@ -43,6 +46,23 @@ jobs:
       - name: Running pub get to fetch dependencies
         run: flutter pub get
+      - name: Check and commit pubspec.lock changes
+        env:
+          HEAD_REF: ${{ github.head_ref }}
+        run: |
+          if [[ $(git status --porcelain pubspec.lock) ]]; then
+            echo "Changes detected in pubspec.lock. Auto-committing updates."
+            git config user.name "github-actions[bot]"
+            git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+            git add pubspec.lock
+            git commit -a -m "chore: Auto-update pubspec.lock in PR #${{ github.event.pull_request.number }}"           
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)